README.md

```diff
@@ -140,6 +140,12 @@ would be useful.
 Signed-off-by: Joe Developer <joe.developer@example.com>
 ```
 
+We take advantage of a change log file to keep track of what was changed in a specific version. We used to handle that file manually by adding entries to it at every pull request. In order to avoid racing issues when pushing multiple PRs, we started to use versionist which will generate the change log at every release. This tool uses two footers from commit log: `Change-type` and `Changelog-entry`. Each PR needs to have at least one commit which will specify both of these two commit log footers. In this way, when a new release is handled, the next version will be computed based on `Change-type` and the entries in the change log file will be generated based on `Changelog-entry`.
```

Comment:
Can we specify the accepted values for Change-type or at least link to somewhere else that defines the accepted values?